source/client/options_impl.cc

```diff
@@ -136,12 +136,19 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       "", "max-active-requests",
       fmt::format("Max active requests (default: {}).", max_active_requests_), false, 0, "uint32_t",
       cmd);
-
+  // NOLINTNEXTLINE
```

Comment:
Is this needed?

Reply:
See https://github.com/envoyproxy/nighthawk/pull/94#issuecomment-508580132
This is annoying, but It would be great if someone can confirm this is a benign issue and I'm not squelching an actual problem that I failed to spot.